### PR TITLE
fix(ingest/db2): fix handling for table properties

### PR DESCRIPTION
**Pull request by deepgarg-visa**
Fixes https://github.com/datahub-project/datahub/issues/8835

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

issue link: https://github.com/datahub-project/datahub/issues/8835

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py [hunk 1]
```diff
@@ -784,7 +786,7 @@ def get_table_properties(
             table_info: dict = inspector.get_table_comment(table, f'"{schema}"')  # type: ignore
 
         description = table_info.get("text")
-        if type(description) is tuple:
+        if type(description) is LegacyRow:
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py:789]:
```suggestion
        if isinstance(description, LegacyRow):
```

**Review by hsheth2 — APPROVED** [review]

**Comment by hsheth2** [comment]
@deepgarg-visa looks like there's some lint issues

**Comment by deepgarg-visa** [comment]
> @deepgarg-visa looks like there's some lint issues

@hsheth2 resolved the lint issues. kindly review

**Review by hsheth2 — APPROVED** [review]

**Comment by deepgarg-visa** [comment]
@hsheth2 
two integration tests are failing which seems to be unrelated
one is related to mssql ingestion and another one is related to vertica ingestion

**Pull request merged**